std_msgs/Header header
std_msgs/Int32 gestureRight
std_msgs/Int32 gestureLeft